README.md

```diff
@@ -8,13 +8,38 @@ This library currently uses the `SHAKE128` ShakeHash with output size of 32 byte
 The signature scheme is `Ed25519` signature algorithm.
 
 ### Usage
-Initiate the tree
+Initiate the tree & history
 ```
 // using DefaultPolicies as current policy of the tree
 m := merkletree.InitMerkleTree(&DefaultPolicies{}, "nonce", "salt")
 // insert existing data (e.g., from db)
 m.Set(key, value)
 ...
+// recompute tree hash
+m.RecomputeHash()
+// generate private key for STR signing
+signKey := crypto.GenerateKey()
+// init STR history chain
+history := NewHistory(m, signKey, startupEpoch, epochInterval)
+```
+
+Update tree in each epoch
+```
+// clone the tree
+m = m.Clone()
+// insert new data
+m.Set(key, value)
+...
+// recompute tree hash
+m.RecomputeHash()
+// update STR history chain
+history.UpdateHistory(m, nextEpoch)
```

Comment:
We're still discussing about it. You can join it here: https://github.com/coniks-sys/libmerkleprefixtree-go/pull/8#discussion_r69050418